content/usando-virtualenv-com-virtualenvwrapper.md

```diff
@@ -0,0 +1,62 @@
+Title: Usando virtualenv com virtualenvwrapper
+Slug: usando-virtualenv-com-virtualenvwrapper
+Date: 2015-09-09 17:49
+Tags: python,desenvolvimento
+Author: André Luiz
+Email:  contato@xdvl.info
+Github: dvl
+Site: http://dvl.rocks/
+Twitter: xdvl
+Category: Python
+
+O uso do `virtualenv` é um item quase que obrigatório em qualquer ambiente de desenvolvimento Python, é usado para manter pacotes (biblioteca Python) de diferentes projetos independentes e isolados dos pacotes do sistema.
+
+O que vejo poucas pessoas usando é o `virtualenvwrapper` que visa a resolver vários problemas criados pelo `virtualenv` como a necessidade de se usar sempre `source bin/activate` e os arquivos do `virtualenv` sendo colocados lado-a-lado com os arquivos do seu projeto.
+
+## Instalação
+
+Primeiro vamos a instalação dos pacotes necessários (essa possivelmente deverá ser a ultima vez que você instala algo globalmente no seu sistema):
+
+    $ sudo pip install -H virtualenv virtualenvwrapper
```

Comment:
A opção --no-site-packages foi incorporada, e é a padrão, então o por
padrão virtualenv não acessa o site-packages
Em 01/12/2015 08:01, "Walmir" <notifications@github.com> escreveu:

In content/usando-virtualenv-com-virtualenvwrapper.md
<https://github.com/pythonclub/pythonclub.github.io/pull/210#discussion_r46265034>
:

> +Author: André Luiz
> +Email:  contato@xdvl.info
> +Github: dvl
> +Site: http://dvl.rocks/

> +Twitter: xdvl
> +Category: Python
> +
> +O uso do `virtualenv` é um item quase que obrigatório em qualquer ambiente de desenvolvimento Python, é usado para manter pacotes (biblioteca Python) de diferentes projetos independentes e isolados dos pacotes do sistema.
> +
> +O que vejo poucas pessoas usando é o `virtualenvwrapper` que visa a resolver vários problemas criados pelo `virtualenv` como a necessidade de se usar sempre `source bin/activate` e os arquivos do `virtualenv` sendo colocados lado-a-lado com os arquivos do seu projeto.
> +
> +## Instalação
> +
> +Primeiro vamos a instalação dos pacotes necessários (essa possivelmente deverá ser a ultima vez que você instala algo globalmente no seu sistema):
> +
> +    $ sudo pip install -H virtualenv virtualenvwrapper

Não sei se o local está certo...
Poderia incluir nas dicas a opção zerada ?

virtualenv --no-site-packages virtualenvwrapper

Desta forma ele começa o ambiente realmente sem nada instalado. No meu caso
quebrou um galho.

—
Reply to this email directly or view it on GitHub
<https://github.com/pythonclub/pythonclub.github.io/pull/210/files#r46265034>
.